esmvaltool/diag_scripts/arctic_ocean/utils.py

```diff
@@ -0,0 +1,245 @@
+"""APPLICATE/TRR Ocean Diagnostics."""
+import logging
+import os
+import shutil
+
+import cmocean.cm as cmo
+import matplotlib as mpl
+import matplotlib.cm as cm
+import numpy as np
+import palettable
```

Comment:
Installation fails because this module is not found. Please add it to setup.py

Reply:
added to setup.py